### DISCUSSION TITLE
Wiki

### DISCUSSION THREAD

**Discussion by encumber** [quer/the-steam-awards#45] [General]
any update on when the wiki pages will be done as I want to use the GiveSteamAward one but there is nothing in the wiki about it making it hard for me to understand how it works without just sending a bunch of awards to someone (which isn't Ideal)

**Comment by quer** [comment]
It comming slowly, as many of the one that use this project. do not program them self. so i have to make sure, it can be understand by most. how it used. and that take time. Off my free time. 

also as the wiki wil be based on the rework, of the project that is kind of done. In the branch "https://github.com/quer/the-steam-awards/tree/Upgrade-core", where it all, way better performas, and less/more Technical. all most all the modules have been updated, to work whit the rework. so now i can mosty start on the wiki. 


the using is more or less simple, even thouge the code, do a lot off things. the methode under here is more or less all you need to give a award. 
```js 
await GiveAward(RequestCommunity, 3, "76561197990233572", randomReward.reactionid, accountInfo.ProfileEdit.webapi_token);
```
So you can just remore the "1." and "2." part, that will just get the point you have, to ensure you can give the award, and will just select a random one, to give. but by using that methode, you can just hard code, what award to use. 

just look in the module file, there is more info, on what each paremater is, and need. 
https://github.com/quer/the-steam-awards/blob/master/modules/GiveSteamAward.js
